MMR.Randomizer/Randomizer.cs

```diff
@@ -825,6 +827,17 @@ private void RandomizeItems()
 
             AddAllItems(itemPool);
 
+            List<(Item, Item)> PlandoItemCombos = PlandoUtils.GetRandomizedItemPlacements(this.Random, itemPool);
+            if (PlandoItemCombos != null)
+            {
+                foreach ((Item item, Item check) in PlandoItemCombos)
+                {
+                    PlaceItem(item, new List<Item> { check });
```

Comment:
I didn't play your april fools, but going to assume you mean plando should be allowed to place items and ignore logic during placement, which doesn't happen since we use PlaceItem, although I can attempt to add.

Reply:
Well the thing is that the placements of items I selected were perfectly in logic, but the algorithm didn't let me place them that way because of various shortcuts and optimizations that it uses. Not sure what the best way to handle this would be.

Reply:
I might need specific examples, so far the only one I've found organically is that bottle of gold dust doesnt like being put early in doggy race, I'm guessing because it doesn't check if you can logically get to it, and so far this logic-bypass seems to get around that example.